core/src/commands/create/create-project.ts

```diff
@@ -145,6 +146,7 @@ export class CreateProjectCommand extends Command<CreateProjectArgs, CreateProje
         presetValues: {
           kind: "Project",
           name,
+          apiVersion: DEFAULT_API_VERSION, // set apiVersion to default value
```

Comment:
```suggestion
          apiVersion: DEFAULT_API_VERSION,
```

I don't think that comment is needed